blocks/components/editable/index.js

```diff
@@ -154,6 +172,22 @@ export default class Editable extends wp.element.Component {
 		}
 	}
 
+	componentWillReceiveProps( nextProps ) {
+		forEach( nextProps.formats, ( state, format ) => {
+			const currentState = this.formats[ format ] || false;
+
+			if ( state !== currentState ) {
```

Comment:
It definitely makes sense to make a habit of this. Thanks for elaborating @aduth!